pkg/db/src/implementations/version-1.2/customer-orders.ts

```diff
@@ -62,6 +68,28 @@ class CustomerOrder implements CustomerOrderInterface {
 		return this;
 	}
 
+	private update(ctx: debug.DebugCtx, data: Partial<CustomerOrderInterface>) {
+		return runAfterCondition(async () => {
+			debug.log(ctx, "customerOrder:update")({ data });
+
+			// Committed notes cannot be updated.
+			if (!this.draft) {
+				debug.log(ctx, "customerOrder:update:customerOrder_not_draft")(this);
+				return this;
+			}
```

Comment:
Shouldn't there be an error?
If the notes are handled in the same way as this, then ok...no error. Let's keep things consistent and, if need be, update notes as well as orders (in case we want to throw an error).
However, if notes do throw an error in this case, yeah...let's keep things consistent.